### DISCUSSION TITLE
typescript-vue-apollo in monorepo with multiple clients?

### DISCUSSION THREAD

**Discussion by sschneider-ihre-pvs** [dotansimha/graphql-code-generator#8595] [Q&A]
I think that might be a question for @liaoliaojun . Is it possible to provide an apollo client that is used for the generated operations because since I try to use it in the monorepo, the usequery from the operations file cause an apollo client with id could not be found.

**Comment by liaoliaojun** [comment]
Sorry, I didn't get the point. But is it so?

```
package.json
  "scripts": {
    "codegen": "graphql-codegen --config codegen/yml/index.yml",
    "codegen:auth": "graphql-codegen --config codegen/yml/auth.yml",
    "codegen:timing": "graphql-codegen --config codegen/yml/timing.yml"
  },
```
